src/pages/CategoriesPage/DeleteCategory/DeleteCategoryComponent.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Grid,
+} from '@mui/material';
+import { styles } from '../styles';
+
+interface Props {
+  deleteModalOpen: boolean;
+  onToggleDeleteStatusClick: () => void;
+  onSubmit: () => void;
+}
+
+const DeleteCategoryComponent = (props: Props) => {
+  return (
+    <Dialog open={props.deleteModalOpen}>
+      <div style = {styles.dialogTitleWrapper}>
+        <DialogTitle style={styles.dialogTitle}>
```

Comment:
DialogTitle should be "Delete Category"

Reply:
Name of the DialogTitle has been changed